_posts/platform/app/2000-01-01-ssl.md

```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
 Encrypt certificate will automatically and immediately replace it. Hence your application will
 always be available using HTTPS.
 
+If you're DNS configuration is not valid or if we fail to generate a
+certificate the platform will automatically retry it later. The time between those tries
+is determined using an exponential backoff algorithm (we will wait 1mn for the
```

Comment:
Long parenthesis are not advised. In this case, it is just another sentence, no need for parenthesis